app/views/govuk_publishing_components/components/_big_number.html.erb

```diff
@@ -1,25 +1,21 @@
 <%
   shared_helper = GovukPublishingComponents::Presenters::SharedHelper.new(local_assigns)
+  big_number_helper = GovukPublishingComponents::Presenters::BigNumberHelper.new(local_assigns)
 
-  number ||= nil
+  number ||= false
   label ||= nil
   href ||= nil
   data_attributes ||= nil
   aria ||= nil
   classes = ["gem-c-big-number"]
   classes << shared_helper.get_margin_bottom
 
-  value_classes = ["gem-c-big-number__value"]
-  
-  if label.nil? && href
-    value_classes << "gem-c-big-number__value--decorated"
-  end
-
+  value_classes = big_number_helper.value_classes
 %>
 <% if number %>
   <% big_number_value = capture do %>
     <%= tag.span class: value_classes, data: href ? nil : data_attributes do %>
-      <%= number %>
+      <%= raw(big_number_helper.formatted_number_string) %>
```

Comment:
Rather than raw-ing this, can we set the HTML entity for plus in the helper instead?

Reply:
I'm very uncomfortable with this `raw` but I wasn't sure how else to do it so I'm glad you've got an alternative idea. Could you please ELI5 your proposal?

Reply:
`&plus;` is the entity you want output. If you use the `.sanitize` method instead it'll do it for you: https://api.rubyonrails.org/classes/ActionView/Helpers/SanitizeHelper.html

Reply:
After revisiting this a bit I've changed how it works so now it returns explicit data instead of html. It probably makes more sense not to handle rendering within the helper. The logic in the template is now a lot lighter than it was before I moved this to a helper. Let me know what you reckon.